loqusdbapi/models.py

```diff
@@ -0,0 +1,80 @@
+from pathlib import Path
+from typing import Optional, List, Union, Any
+
+from pydantic import BaseModel, validator, Field
+
+
+class Individual(BaseModel):
+    ind_id: str
+    case_id: str
+    mother: Optional[str]
+    father: Optional[str]
+    sex: Optional[int] = 0
+    phenotype: Optional[str]
+    ind_index: Optional[int]
+    profile: Optional[list] = []
+    similar_samples: Optional[list] = []
+
+
+class Case(BaseModel):
+    id: Optional[Any] = Field(alias="_id")
+    case_id: str
+    profile_path: Optional[Union[Path, str]]
+    vcf_path: Optional[Union[Path, str]]
+    vcf_sv_path: Optional[Union[Path, str]]
+    nr_variants: Optional[int] = 0
+    nr_sv_variants: Optional[int] = 0
+    individuals: Optional[list] = []
+    sv_individuals: Optional[list] = []
+    inds: Optional[dict] = Field(alias="_inds", default={})
```

Comment:
what is inds? Can the vareable name be more informative?

Reply:
This is a copy of individuals field (i dont know why its there, but it has to be now because database contains it)